Packs/Core/Playbooks/playbook-Netcat_Makes_or_Get_Connections.yml

```diff
@@ -0,0 +1,955 @@
+id: Netcat makes or gets connections
+version: -1
+name: Netcat makes or gets connections
```

Comment:
I think we should consider a different name

Reply:
This is the name according to the excel